src/flows/context/query.tsx

```diff
@@ -83,6 +85,34 @@ const _walk = (node, test, acc = []) => {
   return acc
 }
 
+export const updateBucketInAST = (ast: File, name: string) => {
+  return _walk(
+    ast,
+    node =>
+      node?.type === 'CallExpression' &&
+      node?.callee?.type === 'Identifier' &&
+      node?.callee?.name === 'from' &&
+      node?.arguments[0]?.properties[0]?.key?.name === 'bucket'
+  ).map(node =>
+    lodashSet(
+      node,
+      'arguments.0.properties.0.value.location.source',
+      `"${name}"`
+    )
+  )
+}
+
+export const getBucketsFromAST = (ast: File) => {
+  return _walk(
+    ast,
+    node =>
+      node?.type === 'CallExpression' &&
+      node?.callee?.type === 'Identifier' &&
+      node?.callee?.name === 'from' &&
+      node?.arguments[0]?.properties[0]?.key?.name === 'bucket'
+  ).map(node => lodashGet(node, 'arguments.0.properties.0.value.value', ''))
```

Comment:
We don't use lodash for this since typescript handles it with the obj?.prop?.val format